packages/client/test/cli/cli.spec.ts

```diff
@@ -47,7 +43,7 @@ describe('[CLI]', () => {
           message.includes('network=sepolia chainId=11155111'),
           'client is using custom inputs for network and network ID'
         )
-        child.kill(9)
+        child.kill(15)
```

Comment:
Why does this exit code change?

Reply:
Probably didn't have to.  I was trying to figure out why the tests were hanging.  Will change back.